tidb-cloud/tidb-cloud-billing-recovery-group.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Recovery Group Billing
+summary: Learn about billing for recovery groups in TiDB Cloud.
+---
+
+# Recovery Group Billing
+
+TiDB Cloud bills for recovery groups based on the deployed size of your TiKV nodes in the primary cluster of the recovery group. When you [create a recovery group](/tidb-cloud/recovery-group-get-started.md) for a cluster, you can select the primary cluster for the recovery group. The larger the TiKV configuration, the higher the cost for recovery group protection.
```

Comment:
pls add data processing part

Reply:
Added in https://github.com/pingcap/docs/pull/17425/commits/4eb89d97980e7babd25e5577794c22ef6665f406

Reply:
I think we should not tell the customer how we make the pricing of data processing  - “This data processing cost includes the cross-region and cross-AZ traffic charges”, we just tell them there is a data processing cost charge by GB is enough.

Reply:
Okay, updated in https://github.com/pingcap/docs/pull/17425/commits/6bd256f1a842e600e42f1266960a32ca3307a898